### PR TITLE
.gitignore update

### PR DESCRIPTION
**Pull request by haplav**

### CODE REVIEW

**Comment by haplav** [comment]
@jkruzik I tested the effects but maybe an independent check from you would be welcome. If you are fine with the changes, please squash&merge.

File: .gitignore [hunk 1]
```diff
@@ -35,10 +32,15 @@ GPATH
...
-examples/jbearing2
+/src/**/tests/**/ex[0-9]*
+/src/**/tutorials/**/ex[0-9]*
+/src/tutorials/jbearing2
 *.h.html
 *.c.html
 makefile.html
 index.html
+/arch-*/
+/lib/
```
Comment (jkruzik) [inline .gitignore]: This excludes any additions to lib/permon/
```suggestion
/lib/*
!/lib/permon/
/lib/permon/conf/files
```

Comment (haplav) [reply .gitignore]: Oh gotcha, I forgot there some non-generated stuff as well :-) Thanks.

**Comment by haplav** [comment]
@jkruzik If OK otherwise, please squash&merge.

**Pull request merged**